sched/sched/sched_timerexpiration.c

```diff
@@ -238,7 +238,26 @@ static uint32_t nxsched_cpu_scheduler(int cpu, uint32_t ticks,
     {
       /* Apply the keep alive hack */
 
+#ifdef CONFIG_SCHED_RR_IDLETASK
       return KEEP_ALIVE_TICKS;
+#else
+
+      /* The idle task is the only task in that sched_priority == 0.
+       */
+
+      if (ntcb->sched_priority != 0)
```

Comment:
```suggestion
      if (ntcb->flink != NULL)
```
I think we should use similar approach to one that is used in `bool sched_idletask(void)`

Reply:
let's call sched_idletask instead?

Reply:
Do same check similar to sched_idletask is good idea, I will fix it later